sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1116,9 +1118,289 @@ def max_deflection(self):
         else:
             return None
 
+    def plot_shear_force(self, subs={}):
+        """
+        Returns a plot for Shear force present in the Beam object.
+
+        Parameters
+        ==========
+        subs : dictionary
+            Python dictionary containing Symbols as key and their
+            corresponding values.
+
+        Examples
+        ========
+        There is a beam of length 8 meters. A constant distributed load of 10 KN/m
+        is applied from half of the beam till the end. There are two simple supports
+        below the beam, one at the starting point and another at the ending point
+        of the beam. A pointload of magnitude 5 KN is also applied from top of the
+        beam, at a distance of 4 meters from the starting point.
+        Take E = 200 GPa and I = 400*(10**-6) meter**4.
 
+        Using the sign convention of downwards forces being positive.
 
-class Beam_3d(Beam):
+        >>> from sympy.physics.continuum_mechanics.beam import Beam
+        >>> from sympy import symbols
+        >>> R1, R2 = symbols('R1, R2')
+        >>> b = Beam(8, 200*(10**9), 400*(10**-6))
+        >>> b.apply_load(5000, 2, -1)
+        >>> b.apply_load(R1, 0, -1)
+        >>> b.apply_load(R2, 8, -1)
+        >>> b.apply_load(10000, 4, 0, end=8)
+        >>> b.bc_deflection = [(0, 0), (8, 0)]
+        >>> b.solve_for_reaction_loads(R1, R2)
+        >>> b.plot_shear_force()
```

Comment:
What happens if you call the plot methods and the beam has not been solved yet? i.e. you've only done:

```python
b = Beam()
b.plot_shear_force()
```

Reply:
It returns only co-ordinate axes without a plot as there is no load present on the beam.

Reply:
Ok that's fine.